Enable/View Controllers/EnableBaseViewController.m

```diff
@@ -77,6 +87,18 @@ - (void)testInternetConnection {
     [self.internetReachable startNotifier];
 }
 
+#pragma mark - Theme
+- (void) setupMainTheme {
+    NSDictionary * colorSet = [ThemeTracker sharedTheme].colorSet;
+    [self.activityView setColor:[UIColor colorNamed: colorSet[@"Label"]]];
+    [self.navigationController.navigationBar setBarTintColor:[UIColor colorNamed:colorSet[@"Secondary"]]];
+    [self.navigationController.navigationBar setTintColor:[UIColor colorNamed: colorSet[@"Accent"]]];
+    [self.navigationController.navigationBar setTitleTextAttributes: [NSDictionary dictionaryWithObjects:@[[UIColor colorNamed: colorSet[@"Label"]]] forKeys:@[NSForegroundColorAttributeName]]];
+    [self.view setBackgroundColor:[UIColor colorNamed: colorSet[@"Background"]]];
+}
+- (UIStatusBarStyle)preferredStatusBarStyle{
+    return UIStatusBarStyleDarkContent;
```

Comment:
What I meant was this could be either `UIStatusBarStyleDarkContent` or `UIStatusBarStyleLightContent`, just add a value in your theme structure to include a status bar style flag (light vs dark) and then set it up dynamically in your view controller from the theme